.github/workflows/CI 7.0.yml

```diff
@@ -19,7 +19,35 @@ jobs:
       fail-fast: false
       matrix:
         php-version: [ '8.0.2', '8.1', '8.2', '8.3' ]
-        symfony-version: [ '6.0', '6.1', '6.2', '6.3', '6.4', '7.0', '7.1' ]
+        symfony-version: [ '6.0', '6.1', '6.2', '6.3', '6.4', '7.0', '7.1', '7.2', '7.3' ]
+        exclude:
+          # excluded because of "symfony minimum required" PHP version
+          # well it's a stupid solution but it works
```

Comment:
I think you can remove this comment too :)
This is the way to go! Propel does it too like this.